### PR TITLE
(1.16.0, Docs) Update the ZE and zFTP Readmes and Changelogs

### PR DESCRIPTION
**Pull request by IgorCATech**
Signed-off-by: Igor Kazmyr <igor.kazmyr@broadcom.com>

## Proposed changes

Regular update of the Readme and Changelog for the next release (1.16.0)

## Release Notes

Milestone: 1.16.0

Changelog: updated, too

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [x] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by IgorCATech** [comment]
Running `yarn install --frozen-lockfile` didn't help

**Comment by IgorCATech** [comment]
Zowe Docs PR is here https://github.com/zowe/docs-site/pull/1715

File: packages/zowe-explorer-ftp-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## `1.16.0`
+
+- Added the function that closes a connection
```
Comment (crawr) [inline packages/zowe-explorer-ftp-extension/CHANGELOG.md:5]: This is pretty vague for me. What about: "Added the function that closes the connection at the end of each FTP operation" ?

File: packages/zowe-explorer-ftp-extension/README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ Zowe Explorer's FTP extension adds the FTP protocol to the [Zowe Explorer](https
...
 This VS Code extension also serves as a [source code example](https://github.com/zowe/vscode-extension-for-zowe/tree/master/packages/zowe-explorer-ftp-extension) demonstrating how to use the [Zowe Explorer Extensibility API](https://github.com/zowe/vscode-extension-for-zowe/tree/master/packages/zowe-explorer-api) to create VS Code extensions that extend the Zowe Explorer VS Code extensions with new capabilities.
 
-## What's new in 1.15.0
+## What's new in 1.16.0
 
-Enhancements:
+Added:
 
-- Removed the installation pre-requisite of @zowe/zos-ftp-for-zowe-cli for the FTP Extension for Zowe Explorer
+- Added the function that closes a connection.
```
Comment (crawr) [inline packages/zowe-explorer-ftp-extension/README.md:11]: same here

**Review by crawr — APPROVED** [review]

**Review by lauren-li — APPROVED** [review]
I think this looks good. Thanks @IgorCATech for putting this together!

**Pull request merged**